### PR TITLE
add chart for mongodb-monitoring

### PR DESCRIPTION
**Pull request by simonrondelez**
This PR adds the needed helm config to deploy mongodb-monitoring to k8s

its tested and running

### CODE REVIEW

File: mongodb-monitoring/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    {{- range $key, $val := .Values.prometheus.labels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+  name: "{{ .Release.Name }}"
+  namespace: "{{ .Release.Namespace }}"
+data:
```
Comment (duboisph) [inline mongodb-monitoring/templates/configmap.yaml:13]: So no rules defined yet?

**Review by duboisph — CHANGES_REQUESTED** [review]
You'll need to define a `service` to expose the exporter

File: mongodb-monitoring/templates/configmap.yaml (hunk 1 above)
Comment (simonrondelez) [reply mongodb-monitoring/templates/configmap.yaml:13]: no

File: mongodb-monitoring/templates/_deployment.tpl [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{{ define "mongodb_exporter_deployment" }}
```
Comment (SamClinckspoor) [inline mongodb-monitoring/templates/_deployment.tpl]: this file is a bit weird. you copied it from that other one?
I don't think it will work like this.

Comment (simonrondelez) [reply mongodb-monitoring/templates/_deployment.tpl]: yes copied from the upstream

File: mongodb-monitoring/values.yaml [hunk 3]
```diff
@@ -41,7 +41,7 @@ mongodbExporter:
     ## of a remote MongoDB server. When basic auth is needed,
     ## specify as: <proto>://<user>:<password>@<host>:<port>/<database>. e.g., mongodb://admin:pass@localhost:27017/db.
     ##
-    uri: mongodb://admin:pass@localhost:27017/admin"
+    uri: mongodb://apptweak:vCZEMj3yapVS2DFXR8RF@10.36.30.206:27017/playstore"
```
Comment (SamClinckspoor) [inline mongodb-monitoring/values.yaml]: 👎

File: mongodb-monitoring/templates/configmap.yaml [hunk 4]
```diff
@@ -11,3 +11,36 @@ metadata:
   name: "{{ .Release.Name }}"
   namespace: "{{ .Release.Namespace }}"
 data:
+  mongodb.rules: |-
+    groups:
+    - name: mongoexporter.rules
+      rules:
+      - alert: ElasticsearchExporterDown
```
Comment (SamClinckspoor) [inline mongodb-monitoring/templates/configmap.yaml]: `Elasticsearch`

File: mongodb-monitoring/templates/configmap.yaml (hunk 1 above)
Comment (simonrondelez) [reply mongodb-monitoring/templates/configmap.yaml:13]: now there are

File: mongodb-monitoring/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
```
Comment (SamClinckspoor) [inline mongodb-monitoring/templates/deployment.yaml:1]: `apiVersion: apps/v1beta2` is the latest though not required afaik. `apiVersion: apps/v1` will be in 1.9

**Review by SamClinckspoor — APPROVED** [review]

File: mongodb-monitoring/templates/deployment.yaml (hunk 5 above)
Comment (simonrondelez) [reply mongodb-monitoring/templates/deployment.yaml:1]: tried running it on that version but wasn't working 🤔

Comment (SamClinckspoor) [reply mongodb-monitoring/templates/deployment.yaml:1]: it has a slightly different requirement of the yaml

**Review by duboisph — APPROVED** [review]

**Review by MattiasGees — APPROVED** [review]

**Pull request merged**